### ISSUE TITLE
[ADE Support]: VA Notify : VA Notify Portal: 508 Audit Report Findings - Defect 5 | High

### ISSUE THREAD

**Issue by lakper** [department-of-veterans-affairs/va.gov-team#111567]
> [!NOTE]
> **QUICK SUMMARY:** _This ticket is the child ticket for https://github.com/department-of-veterans-affairs/va.gov-team/issues/110877 . The ticket contains information on all of the defects found by the 508 office in relation to WCAG 1.3.1. Some issues were resolved during the first round of feedback, and they have been crossed out below. The team is looking for guidance on how to resolve the remaining issues (1 in total). Links to the staging site where examples of the issues can be found are listed in the ticket._

## 508 Audit Feedback Received
Defect 5 | High |
_WCAG 2.4.3 Focus Order | If a Web page can be navigated sequentially and the navigation sequences affect meaning or operation, focusable components receive focus in an order that preserves meaning and operability._

Throughout the product, elements do not receive focus in an order that preserves meaning and operability. For example, on the VA Notify screen (Select a service (Accessibility Testing Service 1) -> Home), after navigating to Email or SMS: How to choose and pressing tab twice followed by shift + tab twice, the focus is not returned to Email or SMS: How to choose.

New: 03/25/2025
Not Resolved: 05/28/2025 ❌
***On the Untitled Notification | VA Notify screen (Accessibility Testing Service 1 -> Add Template -> Email), after navigating to the Close Message button, pressing tab followed by shift + tab does not return the focus to the Close Message button.
Location: [https://staging.notifications.va.gov/services/42dd098a-7ad6-4e68-999d-9b0b5ce010ba/templates/new/email/content](https://urldefense.com/v3/__https://staging.notifications.va.gov/services/42dd098a-7ad6-4e68-999d-9b0b5ce010ba/templates/new/email/content__;!!K2n62VMvFw!FxwZ1EPdR6LXeJQPKjd4spootk8jry8DAd6PSt0Q8seiOX4UJEEvalQGKqPlBp5NhHbpWsEtj2Wj-P-6TbI-AMOPoto$)
This is related to a known bug where the banner on the page disappears if the user is navigating with tab and goes past the banner that contains the “Close Message” button. Our proposed solution is to have the banner remain at all times which would allow the user to return focus to the “Close Message” button.

~~Throughout the product, some elements do not receive focus in an order that preserves meaning and operability. For example, on the VA Notify screen (Select a service (Accessibility Testing Service 1) -> Add Template -> Email), tab cannot be used to move the focus to Cancel.~~

~~New: 03/25/2025~~
~~Resolved: 05/28/2025 ✅~~

### Ticket Updates
- [ ] _Connect to an `Epic/Parent` or `Intake` (what body of work is this a part of?) If this is the `Epic/Parent`, connect related tickets as sub-issues._
- [ ] _If this is the `Epic/Parent` ticket, add label for `ADE 508 RECHECK`_
- [ ] _If this ticket is a `Child` ticket aka `sub-issue` please add the appropriate labels for `ADE 508 FLAG CRITICAL`;`ADE 508 FLAG HIGH`; `ADE 508 FLAG MEDIUM` or `ADE 508 FLAG LOW`_
- [ ] _Label with `Originator/Team` (product team or stakeholder requesting support; ADE if internal)_
- [ ] _Label date in the `Open Date` field_
- [ ] _Label with `Estimate` (level of effort expected for this ticket)_
- [ ] _Add `Assignee(s)` name(s) to ticket_ (include Lakwi)
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [ ] _Select a `Priority Level`_
- [ ] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`
- [ ] _Make sure the `ADE 508 RECHECK` label is on this ticket upon closing and that a follow-up date has been established and calendared._

**Comment by coforma-jamie** [comment]
[Findings comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/110877#issuecomment-2959910475)
